__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     'name': "Approvals Statement",
     'version': '1.3',
-    'depends': ['account_accountant'],
+    'depends': ['account_accountant', 'sale_stock'],
```

Comment:
@sbuhl un soucis avec ce depends? la db a le module de toute facon?

Reply:
@shortcut1337 No objection to use it.